app/components/alert_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="<%= classes.join(' ') %>" role="<%= role %>">
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">
+      <%= message %>
+    </p>
+  </div>
+</div>
```

Comment:
is the plan to distribute this same ERB template to Jekyll as well?

Reply:
Yep 👍